packages/ra-ui-materialui/src/list/pagination/Pagination.tsx

```diff
@@ -147,10 +147,47 @@ export const Pagination: FC<PaginationProps> = memo(props => {
     );
 });
 
+type rowsPerPageOptions = {
+    label: string;
+    value: number;
+};
```

Comment:
```suggestion
type RowsPerPageOption = {
    label: string;
    value: number;
};
```

Please use capital casing for types